user/urls.py

```diff
@@ -1 +1,14 @@
-# write your code here
+from django.contrib.auth.views import LoginView, LogoutView
+from django.urls import path
+
+from rest_framework.authtoken import views
+
+from user.views import CreateUserView, LoginUserView, ManageUserView
+
+urlpatterns = [
+    path("register/", CreateUserView.as_view(), name="create"),
+    path("login/", LoginUserView.as_view(), name="login"),
+    path("me/", ManageUserView.as_view(), name="manage"),
```

Comment:
Ensure that the `ManageUserView` is implemented and correctly handles requests to the `me/` endpoint. This is crucial for user management functionality.